README.md

```diff
@@ -28,7 +28,7 @@ Once you have completed these steps you should be able to:
 * Run `pytest` from the room of the project
 * Run `pytest` from `tests/`
 * Run `main.py` from anywhere (e.g. `python src/python_structure/main.py`)
-
+* Run `pylint` on *.py files
```

Comment:
Can you give more detail here?  If we have files in `src/cs334demo` and files in `tests`, how do we get `pylint` to look at all these files?  

Regarding our hallway conversation: how do we specify that pylint should only consider files that contain Python code (generally .py files)?

Finally, @20ivese should not merge this PR.  You are on the same team as @Anthonyjpe.  This should be reviewed by someone not on the team because he/she adds a different context.